src/components/breadcrumb/SubgraphBreadcrumbItem.vue

```diff
@@ -47,7 +47,7 @@
 
 <script setup lang="ts">
 import InputText from 'primevue/inputtext'
-import Menu, { MenuState } from 'primevue/menu'
+import Menu, { type MenuState } from 'primevue/menu'
```

Comment:
Want to also turn on [this linter option](https://typescript-eslint.io/rules/consistent-type-imports)?

Reply:
Maybe [this one](https://typescript-eslint.io/rules/no-import-type-side-effects) too